database/upgrades/17-refactor-sqlite.sql

```diff
@@ -50,6 +50,10 @@ CREATE TABLE reaction_new (
 );
 
 
+DELETE FROM message
+    WHERE (sender, timestamp, part_index, signal_receiver)
+    IN (SELECT DISTINCT sender, timestamp, part_index, signal_receiver FROM message GROUP BY (sender, timestamp, part_index, signal_receiver) HAVING COUNT(*)>1);
```

Comment:
SQLite doesn't seem to like the parentheses around the group by columns. Also, I think the `DISTINCT` is redundant because it's grouped by the same columns?